src/OpenTelemetry.Extensions.Hosting/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@
 * Removes upper constraint for Microsoft.Extensions.Hosting.Abstractions
   dependency. ([#2179](https://github.com/open-telemetry/opentelemetry-dotnet/pull/2179))
 
+* Added `AddOpenTelemetryMetrics` `IServiceCollection` extensions and added
+  `AddInstrumentation<T>`, `AddReader<T>`, & `Configure` extensions to support
```

Comment:
This reads a bit weird, consider rephrase it.

Reply:
Updated, LMK thoughts on the new version